tests/x11/vlc.pm

```diff
@@ -23,7 +23,7 @@ use testapi;
 
 sub run {
     ensure_installed('vlc');
-    x11_start_program('vlc --no-autoscale', target_match => 'vlc-first-time-wizard', match_typed => 'target_match_vlc');
+    x11_start_program('vlc --no-autoscale');
```

Comment:
Without a "target_match" parameter you would have two candidates which look very weird, see https://openqa.opensuse.org/tests/1092390#step/vlc/22

Both parameters have been added for a reason. It would be naive to assume that they are there for no reason. Please read how `x11_start_program` works. I doubt the parameter `target_match` is making any problem.

With this I must say that unfortunately the suggestion in https://progress.opensuse.org/issues/47039#note-7 is not a good one.

Reply:
About target match I totally agree, the other one should not be needed though